cmd/pulumi-test-language/tests/testdata/l1-builtin-try/main.pp

```diff
@@ -1,5 +1,12 @@
 str = "str"
 
+aList = ["a", "b", "c"]
+
+# This should use the try function, not tryOutput.
+output "nonOutputTry" {
+  value = try(aList[0], "fallback")
```

Comment:
Be good to do two versions of this. One that returns list[0] and another that tries to do list[3] and so returns fallback.